README.md

```diff
@@ -23,6 +23,7 @@ A list of resources connected to the [SpaceTraders API](https://spacetraders.io/
 
 * [space_trader](https://github.com/HOWZ1T/space_trader) - An Golang api wrapper with internal caching and an event system.
 * [SpaceMongers](https://github.com/ericgroom/space_mongers) - Elixir API wrapper with integrated rate limiting.
+* [Spacetraders.io](https://github.com/notVitaliy/spacetraders-io) - A Javascript/Typescript SDK
```

Comment:
Two comments, first of a simple fix. You need to end the short description with a period.

Secondly, when looking at the linked repository the installation guide says to install `spacetraders-sdk`. Therefore, I would assume is the name of the project is `spacetraders-sdk` and not `Spacetraders.io`. This is even though it doesn't match with the repository name. Furthermore, using `Spacetraders.io` as the name might lead people on believing it's an official SDK provided by [spacetraders.io](https://spacetraders.io). This might be me being too pedantic, what does @HamishWHC and @HOWZ1T think? Other peoples views are considered as well!

Reply:
I'd agree on the title being weird. spacetraders-sdk would be better I think, for both those reasons.

Reply:
@Thaurin  would you mind updating the second point as well?

Reply:
I agree with @jadlers points.